scripts/control_device.py

```diff
@@ -240,15 +240,15 @@
     choices=possible_swing.keys(),
     required=False,
     help=f"thermostat swing, possible values: {possible_swing}",
-    default=None,
+    default="",
 )
 control_thermostat_parser.add_argument(
     "-t",
     "--temperature",
     type=int,
     required=False,
     help=f"thermostat temperature, possible values: {possible_swing}",
-    default=None,
+    default=0,
```

Comment:
temperature defaulting to 0?
that's a bit extreme, isn't it? do most thermostats even support this?

Reply:
its the default value of int the same way the default value for string is the empty string.

in the control_thermostat we ignore zero value.

and if a lower number than the minimum temperature is provided we set the minimum temperature supported by the AC, same thing for maximum temperature

Reply:
> its the default value of int the same way the default value for string is the empty string.

you don't say... :wink:
note that for the above properties you are referencing with this note,
you shouldn't have used an empty string, to begin with. :upside_down_face:

> in the control_thermostat we ignore zero value.

why 0? have you ever seen an AC unit with a temperature lower than 16 (15 to be safe)?

> and if a lower number than the minimum temperature is provided we set the minimum temperature supported by the AC, same thing for maximum temperature

given https://github.com/TomerFi/aioswitcher/pull/492#discussion_r912333604
why not just remove the "default" property, and set the "required" one to true?
we can even go the extra mile and note the "zero/<15 = max conversation".